src/Illuminate/Database/Schema/Columns/CharString.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Illuminate\Database\Schema\Columns;
+
+/**
+ * Class CharString
+ *
+ * @property-read int $length
+ */
+class CharString extends Text
+{
+    /**
+     * @var int
+     */
+    protected $length;
+
+    /**
+     * CharString constructor.
+     *
+     * @param string $type
+     * @param string $name
+     * @param int $length
+     */
+    public function __construct(string $type, string $name, int $length)
```

Comment:
Please, don't use PHP 7 type hints, is hard to fix conflicts :+1:

Reply:
How is it hard when this is a brand new class, what's it going to conflict with?  😂

Reply:
Might as well follow the rest of the framework code tbh

Reply:
I see PHP 7 is requirement now. Should scalar types hints be used? I think this is great feature and helps a lot. It is impossible to add them in whole code base but they are still useful.
If it is requirement to avoid usage of scalar type hints I will remove them

Reply:
@carusogabriel could you please help me to understand why it is hard to fix conflicts in case of scalar type hints are used?